.cascade-code/Chapman.edu/_components/left_nav/left_nav.vtl

```diff
@@ -25,14 +25,25 @@
 ## Builds the header for the left nav
 #macro ( buildLeftNavTitle )
     ## Data
-    #set ( $folderIndexPage = $_.locatePage("${parentDir.path}/index" ) )
+    #if ( $parentDir.path == "/" )
+        ## For folders under the root, the index doesn't need another forward slash
+        #set ( $folderIndexPage = $_.locatePage("${parentDir.path}index" ) )
+        #set ( $titleLink = $folderIndexPage.link )
+        ## This $title variable should be $folderIndexPage.label, but for the home page
+        ## that label is way too long.
+        #set ( $leftNavTitle = "Chapman University" )
```

Comment:
Discussed this with Ross. The value here is technically supposed to be "Chapman University | A Top Private University in California" because that's the display name for the home page.

That's just a bit too wordy, so Ross said "Chapman University" for the title if the parent of the current page is root.